drivers/sensor/bmp388/Kconfig

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2020 Facebook, Inc. and its affiliates
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BMP388
+	bool "Bosch BMP388 pressure sensor"
+	depends on SPI || I2C
+	help
+	  Enable driver for the Bosch BMP388 pressure sensor
+
+if BMP388
+
+choice BMP388_TRIGGER_MODE
+	prompt "Trigger mode"
+	default BMP388_TRIGGER_GLOBAL_THREAD
+	help
+	  Specify the type of triggering to be used by the driver.
+
+config BMP388_TRIGGER_NONE
+	bool "No trigger"
+
+config BMP388_TRIGGER_GLOBAL_THREAD
+	bool "Use global thread"
+	select BMP388_TRIGGER
+
+config BMP388_TRIGGER_OWN_THREAD
+	bool "Use own thread"
+	select BMP388_TRIGGER
+
+config BMP388_TRIGGER_DIRECT
+	bool "Use IRQ handler"
+	select BMP388_TRIGGER
+endchoice
+
+config BMP388_TRIGGER
+	bool
+
+config BMP388_THREAD_PRIORITY
+	int "Own thread priority"
+	depends on BMP388_TRIGGER_OWN_THREAD
+	default 10
+	help
+	  Priority of the thread used by the driver to handle interrupts.
+
+config BMP388_THREAD_STACK_SIZE
+	int "Own thread stack size"
+	depends on BMP388_TRIGGER_OWN_THREAD
+	default 1024
+	help
+	  Stack size of thread used by the driver to handle interrupts.
+
+choice
+	prompt "Sampling frequency."
+	default BMP388_ODR_RUNTIME
+
+config BMP388_ODR_RUNTIME
+	bool "Change at runtime."
+
+config BMP388_ODR_25_32
+	bool "0.78 Hz"
+
+config BMP388_ODR_25_16
+	bool "1.56 Hz"
+
+config BMP388_ODR_25_8
+	bool "3.125 Hz"
+
+config BMP388_ODR_25_4
+	bool "6.25 Hz"
+
+config BMP388_ODR_25_2
+	bool "12.5 Hz"
+
+config BMP388_ODR_25
+	bool "25 Hz"
+
+config BMP388_ODR_50
+	bool "50 Hz"
+
+config BMP388_ODR_100
+	bool "100 Hz"
+
+config BMP388_ODR_200
+	bool "200 Hz"
+endchoice
+
+config BMP388_OSR_RUNTIME
+	bool "Change at runtime."
+	default y
+
+choice
+	prompt "Pressure oversampling value"
+	default BMP388_OSR_PRESSURE_4
+
+config BMP388_OSR_PRESSURE_1
+	bool "1 sample, 16-bit, 2.64 Pa"
+
+config BMP388_OSR_PRESSURE_2
+	bool "2 samples, 17-bit, 1.32 Pa"
+
+config BMP388_OSR_PRESSURE_4
+	bool "4 samples, 18-bit, 0.66 Pa"
+
+config BMP388_OSR_PRESSURE_8
+	bool "8 samples, 19-bit, 0.33 Pa"
+
+config BMP388_OSR_PRESSURE_16
+	bool "16 samples, 20-bit, 0.17 Pa"
+
+config BMP388_OSR_PRESSURE_32
+	bool "32 Samples, 21-bit, 0.085 Pa"
+endchoice
+
+
+choice
+	prompt "Temperature oversampling value"
+	default BMP388_OSR_TEMP_1
+
+config BMP388_OSR_TEMP_1
+	bool "1 sample, 16-bit, .0050 C"
+
+config BMP388_OSR_TEMP_2
+	bool "2 samples, 17-bit, .0025 C"
+
+config BMP388_OSR_TEMP_4
+	bool "4 samples, 18-bit, .0012 C"
+
+config BMP388_OSR_TEMP_8
+	bool "8 samples, 19-bit, .0006 C"
+
+config BMP388_OSR_TEMP_16
+	bool "16 samples, 20-bit, .0003 C"
+
+config BMP388_OSR_TEMP_32
+	bool "32 Samples, 21-bit, .00015 C"
+endchoice
+
+endif # BMP388
```

Comment:
This all looks like it belongs in DT.

Reply:
I think that at least odr frequency should be put in DT.

Reply:
I've made these all DT properties as requested.